dace/sdfg/work_depth_analysis/work_depth.py

```diff
@@ -282,19 +264,37 @@ def tasklet_depth(tasklet_node, state):
 
 
 def get_tasklet_work(node, state):
-    return tasklet_work(node, state), -1
+    return sp.sympify(tasklet_work(node, state)), sp.sympify(-1)
```

Comment:
There's a ton of these `sympify` calls all over the place in this file. Are these really necessary? Can Sympy not handle it if it has to deal with, say, an integer `0` or `-1`, or a string?